package.json

```diff
@@ -9,7 +9,7 @@
     "browserify": "browserify index.js -s memoizeFormatConstructor > tmp/intl-format-cache.js",
     "test:saucelabs": "npm run browserify && grunt saucelabs",
     "test:unit": "mocha --require test/setup/node-unit --recursive test/unit/",
-    "test": "npm run test:unit",
+    "test": "grunt && npm run test:unit",
```

Comment:
Without this, subsequent test runs will use stale built files.

Reply:
Is this unnecessary for `npm run travis` script? We use it for CI building and testing.

Reply:
Unnecessary as travis always runs clean.